staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -40,6 +40,16 @@ var (
 		[]string{"verb", "host"},
 	)
 
+	requestSize = k8smetrics.NewHistogramVec(
+		&k8smetrics.HistogramOpts{
+			Name: "rest_client_request_size_bytes",
+			Help: "Request size in bytes. Broken down by verb and URL.",
+			// Use buckets ranging from 1000 bytes (1KB) to 10^9 bytes (1GB).
+			Buckets: k8smetrics.ExponentialBuckets(1000, 10.0, 7),
```

Comment:
the buckets are ok?, maybe for client-go we should have lower values?

Reply:
This is fine to me.

Reply:
those are very high values, I changed for other more commons values